src/filesystem/drive/mod.rs

```diff
@@ -243,6 +243,11 @@ impl Drive {
 
         Ok(root_cid)
     }
+
+    pub async fn verifying_keys(&self) -> Vec<VerifyingKey> {
+        let inner = self.inner.read().await;
+        inner.access().verifying_keys()
+    }
```

Comment:
This is probably not quite sufficient. I'd guess you'll want to return a tuple of `(VerifyingKeys, AccessMask)`. When we push these keys to the API we'll want to filter out any historical keys but we don't want to entirely hide those keys from this API so it should be done by the caller.